apps/freelance/src/pages/Freelancer/SearchWork/index.tsx

```diff
@@ -0,0 +1,380 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useForm, SubmitHandler, Controller, ChangeHandler } from "react-hook-form";
+import {
+	StyledPage,
+	Form,
+	InputContainer,
+	Wrapper,
+	ButtonsContainer,
+	InputHeader,
+	InputWrapper,
+	SelectElement,
+} from "./style";
+import {
+	countries,
+	categories,
+	skills,
+	employmentType,
+	hourRate,
+	hoursAmount,
+	workExperience,
+	englishLevel,
+	SelectOptions,
+} from "utils/select-options/options";
+ // hardcoded  - get jobs from server not provided yet
+import { jobs } from "utils/jobs/jobs";
+import { Dashboard, StyledTitle, StyledButton, JobCard, Pagination } from "@freelance/components";
+import { it } from "node:test";
+import { useGetJobsQuery } from "redux/jobs/jobs.api";
+
+type user = "freelancer" | "employer";
+
+interface IFormInput {
+	category: SelectOptions;
+	position: string;
+	skills: SelectOptions[];
+	employmentType: SelectOptions;
+	englishLevel: SelectOptions;
+	hourRate: SelectOptions;
+	availableAmountOfHour: SelectOptions;
+}
+
+export function SearchWork() {
+	const user: user = "freelancer";
+	const { t } = useTranslation();
+	const { handleSubmit, control, getValues, reset } = useForm<IFormInput>();
+	const { isLoading, isError, data } = useGetJobsQuery();
+	const emptyValue = {
+		value: "",
+		label: "",
+	};
+
+  // hardcoded  - get freelanceer profile not provided yet
+	const freelancerFilter = {
+		position: "",
+		category: "Category",
+		skills: ["JavaScript/Front-End"],
+		employmentType: "",
+		englishLevel: "",
+		hourRate: "",
+		availableAmountOfHour: "",
+	};
+
+	const [filterJobs, setFilterJobs] = useState(jobs);
+	const [filter, setFilter] = useState<IFormInput | any>({
+		category: "",
+		position: "",
+		skills: [],
+		employmentType: "",
+		englishLevel: "",
+		hourRate: "",
+		availableAmountOfHour: "",
+	});
+	const [toggleFilter, setToggleFilter] = useState<string>("reset");
+	const [useFilters, setUseFilters] = useState<boolean>(false);
+
+	let skillsIncludesArr: boolean[] = [];
```

Comment:
NEVER NEVER use constant in component, in initialize with each rerender

Reply:
updated